framework/global/log.h

```diff
@@ -48,5 +48,6 @@ inline QDebug operator<<(QDebug debug, const std::string& s)
 
 #define NOT_IMPLEMENTED LOGE() << "NOT IMPLEMENTED "
 #define NOT_SUPPORTED LOGE() << "NOT SUPPORTED "
+#define UNUSED(x) (void)x;
```

Comment:
There is a weak relation with log output here)
Why can't we use Q_UNUSED or even hide the variable name if it's unused?

Reply:
`log.h` is not just a log output, it is a number of convenient development tools. Of course, they can be scattered across different files, but I see no reason.

This macro has been added so that where it is easy not to use Qt, it should not be used :)